### ISSUE TITLE
Update IDBANK file

### ISSUE THREAD

**Issue by tfardet** [InseeFrLab/pynsee#188]
The IDBANK file does not seem to be valid anymore

```python
from pynsee.macrodata import get_series_list

tourism_data = get_series_list('TOURISME-FRANCE-METHODE-REDRESSEMENT-2019')
```

Complains

```
Package's internal data has been used !
Idbank file download failed, have a look at the following page and find the new link !
https://www.insee.fr/en/information/2868055

You may change the downloaded file changing the following environment variable !
import os; os.environ['pynsee_idbank_file'] = 'my_new_idbank_file'Please contact the package maintainer if this error persists !
```

But the ZIP file does not seem to correspond to the one provided in the linked page.

Maybe providing a function to call to re-generate the file could be useful?

**Comment by hadrilec** [comment]
Hi @tfardet,
the problem is due to the fact that for the last publication INSEE zipped the file twice instead of once as they used to do.
I provided a fix 2 weeks ago in this commit 367cc45 on the master branch.
I will try to find the time to make a new release before the 1st of March.

**Comment by tfardet** [comment]
Ah, that's what that commit was about, I understand, thanks!
